amti/actions/create.py

```diff
@@ -251,7 +316,8 @@ def create_batch(
         client,
         definition_dir,
         data_path,
-        save_dir):
+        save_dir,
+        verify_cost_before_upload):
```

Comment:
Since `verify_cost_before_upload` requires user interaction, it might be better to include this logic in the script rather than the backend actions functions.